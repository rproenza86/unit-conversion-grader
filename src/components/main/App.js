import React from 'react';

// component dependencies
import Header from '../header/Header';
import Footer from '../footer/Footer';
import Grader from '../grader/Grader';

// styles
import './App.scss';

function App() {
    return (
        <div className="App">
            <Header />
            <main className="App-main">
                <Grader />
            </main>
            <Footer />
        </div>
    );
}

export default App;
