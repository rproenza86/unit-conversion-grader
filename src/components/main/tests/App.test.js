import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('Main App component', () => {
    it('should renders without crashing', () => {
        shallow(<App />);
    });
    it('should display render Grader component', () => {
        const component = shallow(<App />);
        const tree = component.getElement();
        expect(tree).toMatchSnapshot();
    });
});
