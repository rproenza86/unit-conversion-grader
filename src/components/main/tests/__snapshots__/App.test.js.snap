// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main App component should display render Grader component 1`] = `
<div
  className="App"
>
  <Header />
  <main
    className="App-main"
  >
    <Grader />
  </main>
  <Footer />
</div>
`;
